package.json

```diff
@@ -1,159 +1,168 @@
+      "import": "./src/basics.js"
+    },
+    "./block": {
+      "types": "./types/src/block.d.ts",
+      "import": "./src/block.js"
+    },
+    "./bytes": {
+      "types": "./types/src/bytes.d.ts",
+      "import": "./src/bytes.js"
+    },
+    "./cid": {
+      "types": "./types/src/cid.d.ts",
+      "import": "./src/cid.js"
+    },
+    "./codecs/json": {
+      "types": "./types/src/codecs/json.d.ts",
+      "import": "./src/codecs/json.js"
+    },
+    "./codecs/raw": {
+      "types": "./types/src/codecs/raw.d.ts",
+      "import": "./src/codecs/raw.js"
     },
     "./hashes/digest": {
+      "types": "./types/src/hashes/digest.d.ts",
       "import": "./src/hashes/digest.js"
     },
+    "./hashes/hasher": {
+      "types": "./types/src/hashes/hasher.d.ts",
+      "import": "./src/hashes/hasher.js"
+    },
+    "./hashes/identity": {
+      "types": "./types/src/hashes/identity.d.ts",
+      "import": "./src/hashes/identity.js"
+    },
     "./hashes/sha2": {
+      "types": "./types/src/hashes/sha2.d.ts",
       "browser": "./src/hashes/sha2-browser.js",
       "import": "./src/hashes/sha2.js"
     },
-    "./hashes/identity": {
-      "import": "./src/hashes/identity.js"
+    "./interface": {
+      "types": "./types/src/interface.d.ts",
+      "import": "./src/interface.js"
     },
-    "./codecs/json": {
-      "import": "./src/codecs/json.js"
+    "./link": {
+      "types": "./types/src/link.d.ts",
+      "import": "./src/link.js"
     },
-    "./codecs/raw": {
-      "import": "./src/codecs/raw.js"
+    "./traversal": {
+      "types": "./types/src/traversal.d.ts",
+      "import": "./src/traversal.js"
     }
   },
-  "devDependencies": {
-    "@ipld/dag-pb": "^2.1.14",
-    "@stablelib/sha256": "^1.0.1",
-    "@stablelib/sha512": "^1.0.1",
-    "@types/chai": "^4.3.0",
-    "@types/chai-as-promised": "^7.1.4",
-    "@types/mocha": "^9.0.0",
-    "@types/node": "^18.0.0",
-    "@typescript-eslint/eslint-plugin": "^5.6.0",
-    "@typescript-eslint/parser": "^5.6.0",
-    "buffer": "^6.0.3",
-    "c8": "^7.10.0",
-    "chai": "^4.3.4",
-    "chai-as-promised": "^7.1.1",
-    "cids": "^1.1.9",
-    "ipjs": "^5.2.0",
-    "mocha": "^10.0.0",
-    "polendina": "^3.0.0",
-    "standard": "^17.0.0",
-    "typescript": "^4.5.4"
-  },
-  "standard": {
-    "ignore": [
-      "dist",
-      "vendor"
-    ]
-  },
-  "directories": {
-    "test": "test"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/multiformats/js-multiformats.git"
-  },
-  "bugs": {
-    "url": "https://github.com/multiformats/js-multiformats/issues"
+  "browser": {
+    "./hashes/sha2": "./src/hashes/sha2-browser.js",
+    "./src/hashes/sha2.js": "./src/hashes/sha2-browser.js"
   },
-  "homepage": "https://github.com/multiformats/js-multiformats#readme",
-  "typesVersions": {
-    "*": {
-      "*": [
-        "types/src/*"
-      ],
-      "types/*": [
-        "types/*"
-      ]
+  "eslintConfig": {
```

Comment:
I would really love if we tap into @hugomrdias's config skills, that are able to reconcile prettier & eslint configs.

I personally hate touching this repo because my editor tends to reformat everything with prettier which makes eslint unhappy so often have to disable prettier and then try to manually fix all the stuff.

@hugomrdias seems to have a way to configure repos such that tells both eslint & prettier how to work together in synergy.

Reply:
these are my confs https://github.com/hugomrdias/hd-scripts ts, eslint, prettier working happily together